libraries/AC_WPNav/AC_Circle.h

```diff
@@ -101,4 +101,5 @@ class AC_Circle
     float       _angular_vel;   // angular velocity in radians/sec
     float       _angular_vel_max;   // maximum velocity in radians/sec
     float       _angular_accel; // angular acceleration in radians/sec/sec
+    AP_Int8     _stopping_point_center_enable;  // The circle to the stopping point
```

Comment:
Nitpicking: change this to be next to the other parameters.

Reply:
Move to next variable _rate.